typing-tests/tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "target": "es6",
```

Comment:
Instead of targeting `es6` (which means native promise support is expected), maybe it would be better to add `"lib": ["es2015"]` (which will make promises work for `es3` and `es5` targets).

Reply:
Agreed. Done.